crates/aptos-faucet/ts-client/src/services/GeneralService.ts

```diff
@@ -0,0 +1,38 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import type { CancelablePromise } from '../core/CancelablePromise';
+import type { BaseHttpRequest } from '../core/BaseHttpRequest';
+
+export class GeneralService {
+
+    constructor(public readonly httpRequest: BaseHttpRequest) {}
+
+    /**
+     * Show OpenAPI explorer
+     * Provides a UI that you can use to explore the API. You can also
+     * retrieve the API directly at `/spec.yaml` and `/spec.json`.
+     * @returns string
+     * @throws ApiError
+     */
+    public spec(): CancelablePromise<string> {
+        return this.httpRequest.request({
+            method: 'GET',
+            url: '/spec',
+        });
+    }
+
+    /**
+     * Check API health
+     * Basic endpoint that always returns Ok for health.
+     * @returns string
+     * @throws ApiError
+     */
+    public root(): CancelablePromise<string> {
+        return this.httpRequest.request({
+            method: 'GET',
+            url: '/',
```

Comment:
why not `/health`?

Reply:
In GCP and other cloud deployments the default health check endpoint is `/` so it made sense.